at_secondary/at_secondary_server/lib/src/notification/at_notification_map.dart

```diff
@@ -32,22 +28,32 @@ class AtNotificationMap {
     _computeWaitTime(atNotification);
   }
 
-  /// Returns the map of first N entries.
-  Iterator<AtNotification> remove(String? atsign) {
-    // If map is empty, return empty map
-    if (_notificationMap.isEmpty) {
-      return [].iterator as Iterator<AtNotification>;
+  int numQueued(String atSign) {
+    // If map is empty, or map doesn't contain the atSign, return an iterator for an empty list
+    if (_notificationMap.isEmpty || !_notificationMap.containsKey(atSign)) {
```

Comment:
since we are returning from if block can we remove else block
e.g
if(_notificationMap.isEmpty || !_notificationMap.containsKey(atSign)) {
return 0;
}
// rest of the code

Reply:
Sounds good, will do. Makes it a little more readable